spec/rails_helper.rb

```diff
@@ -77,6 +77,8 @@
 
 def create_submitter(submitter)
```

Comment:
Not directly related to this PR, but wondering if this function might find a better home in /spec/support.